proposals/lip-0048.md

```diff
@@ -204,12 +205,27 @@ def getFeeTokenID() -> TokenID:
     return TOKEN_ID_FEE
 ```
 
+### Genesis Block Processing
+
+#### Genesis State Finalization
+
+```python
+def finalizeGenesisState(g: Block) -> None:
+    if ADDRESS_FEE_POOL is not None:
+        Token.initializeUserAccount(ADDRESS_FEE_POOL, TOKEN_ID_FEE)
```

Comment:
I think this will fail in the token module, need to clarify how to do this.

Reply:
Noted.